src/App.jsx

```diff
@@ -1,104 +1,32 @@
 import './App.scss';
-
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+function getUsers(userId) {
```

Comment:
```suggestion
function getUserById(userId) {
```
You get only one user and you get by id